Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule_test.py

```diff
@@ -3058,3 +3058,47 @@ def test_add_or_remove_tag_endpoint_command(requests_mock, args, expected_filter
             'tag': 'test'
         }
     }
+
+
+excepted_output_1 = {'filters': [{'field': 'endpoint_status',
+                                  'operator': 'IN', 'value': ['connected']}], 'new_alias_name': 'test'}
+excepted_output_2 = {'filters': [{'field': 'endpoint_status',
+                                  'operator': 'IN', 'value': ['connected']}], 'new_alias_name': ""}
+
+
+@pytest.mark.parametrize('input, expected_output', [("test", excepted_output_1),
+                                                    ('""', excepted_output_2)])
+def test_endpoint_alias_change_command__diffrent_alias_new_names(mocker, input, expected_output):
+    """
+    Given:
+    - command arguments with endpoint filters, in the first case the new alias name is a string of text,
+    and in the second case the new alias name is a string of only double quotes.
```

Comment:
you can just use this format next time, as docstrings are meant to be short and exact:

Given:
    - valid new alias name as string
    - empty new alias name (due to xsoar limitation, represented by a string of double quote)
    
   Then:
       Makes sure the request body is created correctly.